src/elements/input/src/index.tsx

```diff
@@ -1,10 +1,11 @@
 /** @jsx jsx */
 
-import { useState } from 'react'
+import { useEffect, useRef, useState } from 'react'
 import { jsx } from '@emotion/core'
 import { FrozenInput } from '@uswitch/trustyle.frozen-input'
 import { inputs } from '@uswitch/trustyle.styles'
 import * as ReactInputMask from 'react-input-mask'
+import debounce from 'lodash.debounce'
```

Comment:
Is this really needed? Do we really need to add another dependency into trustyle? 
https://www.30secondsofcode.org/snippet/debounce is pretty straightforward to implement.